script.js

```diff
@@ -1,3 +1,7 @@
+// function closeMenu() {
```

Comment:
Please don't leave any commented code in your codebase. In this way, you can keep your codebase clean and organized.